Src/AutoFixture.xUnit.net2/FrozenAttribute.cs

```diff
@@ -110,7 +110,10 @@ private ICustomization FreezeByCriteria(ParameterInfo parameter)
                 .Or(ByParameterName(type, name))
                 .Or(ByFieldName(type, name));
 
-            return new FreezeOnMatchCustomization(type, filter);
+            return new FreezeOnMatchCustomization(type, filter)
```

Comment:
By adding a [new constructor that accepts the `request` as `object`](https://github.com/AutoFixture/AutoFixture/pull/695/files#r75438371), it should be enough to say `return new FreezeOnMatchCustomization(request, filter)`, shouldn't it?